src/SFA.DAS.AssessorService.Application.Api/Controllers/RegisterController.cs

```diff
@@ -93,5 +94,41 @@ public async Task<IActionResult> UpdateEpaOrganisation([FromBody] UpdateEpaOrgan
         }
 
 
+        [HttpPost("standards",Name = "CreateEpaOrganisationStandard")]
+        [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(EpaOrganisationStandard))]
+        [SwaggerResponse((int)HttpStatusCode.NotFound, typeof(ApiResponse))]
+        [SwaggerResponse((int)HttpStatusCode.BadRequest, typeof(ApiResponse))]
+        [SwaggerResponse((int)HttpStatusCode.Conflict, Type = typeof(ApiResponse))]
+        [SwaggerResponse((int)HttpStatusCode.InternalServerError, Type = typeof(ApiResponse))]
+        public async Task<IActionResult> CreateOrganisationStandard([FromBody] CreateEpaOrganisationStandardRequest request)
+        {
+            try
+            {
+                _logger.LogInformation("Creating new Organisation Standard");
+                var result = await _mediator.Send(request);
+                return Ok(new EpaOrganisationStandardResponse(result));
+            }
+            catch (NotFound ex)
+            {
+                _logger.LogError($@"Record is not available for organisation / standard: [{request.OrganisationId}, {request.StandardCode}]");
```

Comment:
There is some middleware we have added - See ErrorWrappingMiddleware which probably could do with extending to try to remove noise in the controllers cause by using try / catch blocks. Might be worthwhile taking a look.